### ISSUE TITLE
Increased circulating interferon alpha concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9026]
**Preferred term label:**
Increased circulating interferon alpha concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
An elevation in the concentration of interferon alpha measured in the blood circulation.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
615010 
 PMID: 23001123   6/7

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This was Increased circulating Interferon-alpha concentration HP:0034513
